### DISCUSSION TITLE
Welcome to seanpm2001 Discussions!

### DISCUSSION THREAD

**Discussion by seanpm2001** [seanpm2001/seanpm2001#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sarrac3873** [comment]
> 
> ## 👋 Welcome!
>   We’re using Discussions as a place to connect with other members of our community. We hope that you:
>   * Ask questions you’re wondering about.
>   * Share ideas.
>   * Engage with other community members.
>   * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
>   To get started, comment below with an introduction of yourself and tell us about what you do with this community.
> 
> 
>

**Reply by sl4shed** [reply]
jigsaw

**Comment by wbreitenstein** [comment]
Wood

**Reply by sl4shed** [reply]
Thas tuff ngl
